src/sentry/static/sentry/app/views/settings/components/forms/tableField.tsx

```diff
@@ -53,7 +41,10 @@ export default class TableField extends React.Component<Props> {
     allowEmpty: PropTypes.bool,
   };
 
-  static defaultProps = defaultProps;
+  static defaultProps = {
```

Comment:
Why move these back in? See https://www.notion.so/sentry/Sentry-Typescript-Tips-cef7ae63b4334156945b93b30a651b14

Reply:
I had a reason but it wasn't a good one, fixed